config/pmd-main.xml

```diff
@@ -16,9 +16,7 @@
             <!-- Fields with these annotations will have their value injected by picocli.
               These fields should not be marked final. -->
             <property name="ignoredAnnotations"
-                      value="picocli.CommandLine.Option
```

Comment:
There should be `$` in the annotation name. And we do not need other annotations except this one.